toolchain/driver/compile_subcommand.cpp

```diff
@@ -479,11 +504,19 @@ class CompilationUnit {
       module_->print(driver_env_->output_stream, /*AAW=*/nullptr,
                      /*ShouldPreserveUseListOrder=*/true);
     }
+    if (timings_) {
+      timings_->Add("lower", end_time - start_time);
+    }
   }
 
   auto RunCodeGen() -> void {
     CARBON_CHECK(module_);
+    auto start_time = std::chrono::steady_clock::now();
     LogCall("CodeGen", [&] { success_ = RunCodeGenHelper(); });
+    auto end_time = std::chrono::steady_clock::now();
+    if (timings_) {
```

Comment:
```suggestion
    if (timings_) {
      auto end_time = std::chrono::steady_clock::now();
```

Minor thing, just not necessary to get the end time if no timings_

Note, I'd be happy if you want to do similar with the above cases, just adding timing immediately where you record `end_time`.